package.json

```diff
@@ -11,43 +11,45 @@
     "axios": ">= 0.18.0",
     "jquery": "^3.3.1",
     "lodash": "^4.17.11",
+    "planout": "git://github.com/rawls238/Planout.js.git#97807c149d01031b972d9c3df7fb93558a166120",
```

Comment:
Any chance we can move onto a normal version of Planout now that you added type definitions?

Reply:
This is the current tip of the normal version, the maintainer has not yet uploaded the new package to npm after merging the PR.
See:
https://github.com/rawls238/PlanOut.js/commits/master 
https://www.npmjs.com/package/planout

Reply:
oh whoops misread that... thought we were using a fork